pwa/src/components/configurationsExportButton/ConfigurationsExportButton.tsx

```diff
@@ -27,19 +27,23 @@ export default function ConfigurationsExportButton() {
         });
       })
       .catch((error) => {
-        throw new Error (error)
+        throw new Error(error)
+      })
+      .finally(() => {
+        setLoading(false)
       })
-      .finally(() => { setLoading(false) })
   };
 
   return (
-    <button
-      className="utrecht-button"
-      type="button"
-      onClick={handleExport}
-      disabled={loading}
-    >
-      {loading ? "Perparing your download...": "Export Configuration"}
-    </button>
+    <div>
```

Comment:
This `<div>` wrapper should be removed. 

I understand why you've added it. But this layout logic should be added where the `ConfigurationsExportButton` is implemented (in this case in `/pages/configurations/index.tsx`).

Reply:
If i <div> remove . The button is not in the right place